ros-conversion/v2x-ros-conversion/CMakeLists.txt

```diff
@@ -15,13 +17,19 @@ cmake_minimum_required(VERSION 3.5)
 project(v2x-ros-conversion)
 
 find_package(carma_cmake_common REQUIRED)
-carma_check_ros_version(2)
 carma_package()
 
+find_package(ros_environment REQUIRED)
```

Comment:
We can remove lines 22 and 24 here we dont need to check ros-version since this is meant for ros2-humble only